packages/root-config/src/applications.js

```diff
@@ -1,8 +1,19 @@
 const PluginImportPathPre = "zuri-plugin-";
 const PluginAppPathPre = "plugin-";
 
+const textAlign = () => {
```

Comment:
also why are you working in the root config???

Reply:
I put a functionality to apply text align right to Arabic and Hebrew texts

Reply:
i've already made the chages please review